### PR TITLE
Fix ineffassign issues

### PR DESCRIPTION
**Pull request by rollandf**
Fix issues found by:
`golangci-lint run --disable-all -E ineffassign`

Signed-off-by: Fred Rolland <frolland@nvidia.com>

### CODE REVIEW

**Comment by rollandf** [comment]
See issue #288

**Review by adrianchiris — APPROVED** [review]

**Review by SchSeba — APPROVED** [review]

**Comment by SchSeba** [comment]
@rollandf can you please rebase?

**Comment by rollandf** [comment]
@SchSeba rebased

File: test/conformance/tests/sriov_operator.go [hunk 1]
```diff
@@ -1970,6 +1971,37 @@ func createSriovPolicy(sriovDevice string, testNode string, numVfs int, resource
 	}, 10*time.Minute, time.Second).Should(Equal(int64(numVfs)))
 }
 
+func createUnschedulableTestPod(node string, networks []string, resourceName string) {
```
Comment (adrianchiris) [inline test/conformance/tests/sriov_operator.go:1974]: @rollandf you added this ?

Comment (rollandf) [reply test/conformance/tests/sriov_operator.go:1974]: Yeah, seems a rebase issue. Fixing.

**Comment by adrianchiris** [comment]
thanks @rollandf  ! :)

**Pull request merged**